### PR TITLE
feat: Add about us page - MEED-1716 - Meeds-io/MIPs#48 

### PR DESCRIPTION
**Pull request by MayTekayaa**
This change will add about us page which contains four parts: 
- Governance
- What we do
- Sustainability
- Founding members

### CODE REVIEW

**Review by boubaker — DISMISSED** [review]

File: deeds-dapp-webapp/src/main/webapp/static/html/about-us.html [hunk 1]
```diff
@@ -1,3 +1,255 @@
-<div>
-  <v-chip>{{ $t('aboutUs') }}</v-chip>
+<div class="d-flex flex-column mt-8 mt-sm-10">
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/static/html/about-us.html:1]: Missing license Header

Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/static/html/about-us.html:1]: added!

File: deeds-dapp-webapp/src/main/webapp/vue-app/staticPage/components/StaticPageContent.vue [hunk 2]
```diff
@@ -18,6 +18,10 @@
 -->
 <script>
 export default {
-  template: document.getElementById('staticContent').innerHTML.replace(/images\//g, `/${window.location.pathname.split('/')[1]}/static/images/`),
+  template: document.getElementById('staticContent').innerHTML,
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/vue-app/staticPage/components/StaticPageContent.vue]: why changing the strategy of images ?

Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/vue-app/staticPage/components/StaticPageContent.vue]: I need the parentLocation to redirect the user to /owners and /portfolio

Comment (boubaker) [reply deeds-dapp-webapp/src/main/webapp/vue-app/staticPage/components/StaticPageContent.vue]: The fact that you need `parentLocation` doesn't mean we need to make adding images in html content, something complex.

File: deeds-dapp-webapp/src/main/webapp/static/html/about-us.html [hunk 3]
```diff
@@ -1,3 +1,270 @@
...
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ -->
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/static/html/about-us.html]:
```suggestion
```
My bad ! Please delete comment to reduce page size in production.

File: deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less [hunk 4]
```diff
@@ -238,6 +239,24 @@ h4 {
...
+}
+
+.logo-m {
+  filter: invert(47%) sepia(46%) saturate(419%) hue-rotate(134deg) brightness(89%) contrast(90%);
+  opacity: 0.5;
+  position: absolute;
+  right: -80px;
+}
+
+#btn-dark-grey:hover {
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less]: 1. id must be unique in page, use class `dark-grey-border-color:hover` instead
2. why using a special effect that is different from standard UX of Meeds package ?
3. if this new UX should be adopted for some buttons, please rename the class to something more understandable.

Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less]: It's new UX adopted for some buttons on dapp
Done!

File: deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less [hunk 5]
```diff
@@ -238,6 +239,24 @@ h4 {
...
+}
+
+.logo-m {
+  filter: invert(47%) sepia(46%) saturate(419%) hue-rotate(134deg) brightness(89%) contrast(90%);
+  opacity: 0.5;
+  position: absolute;
+  right: -80px;
+}
+
+.primary-btn:hover {
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less]: It would better to use buttons with style `<v-btn color="primary">...</v-btn>`

Comment (boubaker) [reply deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less]: should we inherit the same style as Vuetify style of the Meeds package by including the file `vuetify-all.less` and replace the one defined in dApp ? WDYT ?

Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less]: In fact the added class is for the hover effect for the buttons and it's primary color

Comment (boubaker) [reply deeds-dapp-webapp/src/main/webapp/skin/less/deeds.less]: I added in a commit the `vuetify-all.less` inherited from Meeds Package as suggested.

**Review by boubaker — APPROVED** [review]

**Pull request merged**